### PR TITLE
Cleanup 1

### PR DESCRIPTION
**Pull request by triggermike**
Fixed content description warning & cleared text view on nulls

### CODE REVIEW

**Comment by triggermike** [comment]
Content descriptions have changed. See next PR, "Matched button styles".

File: app/src/main/java/com/hugheswd/outtatime/MainActivity.kt [hunk 1]
```diff
@@ -24,10 +24,9 @@ class MainActivity : AppCompatActivity() {
             timerStop()
         }
 
-        etMainUserInput.setOnKeyListener { v, keyCode, event ->
+        etMainUserInput.setOnKeyListener {v, keyCode, event ->
             if(keyCode == KeyEvent.KEYCODE_ENTER && event.action == KeyEvent.ACTION_DOWN){
                 timerStart()
-                true
```
Comment (HTMLlama) [inline app/src/main/java/com/hugheswd/outtatime/MainActivity.kt]: Did you test this to make sure it didn't crash?

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -96,6 +96,7 @@
         android:layout_marginStart="8dp"
         android:layout_marginTop="8dp"
         android:background="@color/colorBackground"
+        android:contentDescription="@null"
```
Comment (HTMLlama) [inline app/src/main/res/layout/activity_main.xml]: We should try to never use a null content description, just put in something generic in the Strings file and add a reference to the string here

File: app/src/main/java/com/hugheswd/outtatime/MainActivity.kt (hunk 1 above)
Comment (triggermike) [reply app/src/main/java/com/hugheswd/outtatime/MainActivity.kt]: I did, no issues

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (triggermike) [reply app/src/main/res/layout/activity_main.xml]: I think I changed all of those and added actual strings in the other commit. I'll double-check

**Pull request merged**